### PR TITLE
editor: page refresh when 412 error responded

### PR DESCRIPTION
**Pull request by oguzdemirbasci**
Ref: #2013

### CODE REVIEW

**Review by drjova — APPROVED** [review]

**Comment by michamos** [comment]
as I've already discussed privately with @oguzdemirbasci on the chat, I don't think this is a good solution:
```quote
I think there's a much broader problem with caching
  13:29

like you need to refresh the page in the UI too after changes for them to be seen
  13:29

maybe there's a unique solution to both problems
  13:29

(I don't know much about browser caching)
  13:30

there's also https://github.com/inspirehep/inspirehep/issues/2012
  13:30

if the editor closes after saving, then they'll need to reopen the editor to make a further change, at which point "fresh" data should be loaded
  13:31

to summarize, I don't think it makes much sense to work on this bug in isolation, it's connected to other things
```
